src/MonticelloConfigurations/MCConfiguration.class.st

```diff
@@ -275,7 +275,8 @@ MCConfiguration >> loadVersions: aCollection [
 
 { #category : #accessing }
 MCConfiguration >> log [
-	^log ifNil: [Transcript]
```

Comment:
is this log method used?

Reply:
I think nobody uses anymore the Monticello Configurations or the MC Browser.
So I could remove it but in another branch so we don't mix changes?